src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2098,6 +2098,18 @@ limitations under the License.
             </include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-network-infrastructure-manager</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Network Infrastructure Manager</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0xFFF10010</deviceId><!-- TODO: ID-TBD -->
```

Comment:
do we have a github or spec issue that could be used to track this?

Reply:
Yes this is tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/8223